src/resources/js/components/layout/SideBar.vue

```diff
@@ -2,19 +2,40 @@
     <v-list
         nav
         dense
+        v-if="isLoaded"
     >
-        <v-list-item
-            v-for="(route, i) in routes"
-            :key="i"
-            :href="route.path"
+        <v-container
+            v-if="isLogged"
         >
-            <v-list-item-icon>
-                <v-icon>{{ route.icon }}</v-icon>
-            </v-list-item-icon>
-            <v-list-item-content>
-                <v-list-item-title v-text="route.name"></v-list-item-title>
-            </v-list-item-content>
-        </v-list-item>
+            <v-list-item
+                v-for="(route, i) in authRoutes"
+                :key="i"
+                :href="route.path"
+            >
+                <v-list-item-icon>
+                    <v-icon>{{ route.icon }}</v-icon>
+                </v-list-item-icon>
+                <v-list-item-content>
+                    <v-list-item-title v-text="route.name"></v-list-item-title>
+                </v-list-item-content>
+            </v-list-item>
+        </v-container>
+        <v-container
+            v-else
+        >
+            <v-list-item
+                v-for="(route, i) in notAuthRoutes"
+                :key="i"
+                :href="route.path"
+            >
+                <v-list-item-icon>
+                    <v-icon>{{ route.icon }}</v-icon>
+                </v-list-item-icon>
+                <v-list-item-content>
+                    <v-list-item-title v-text="route.name"></v-list-item-title>
+                </v-list-item-content>
+            </v-list-item>
```

Comment:
instead of that copy past logic you could simply assing the correct routes to a routes variable on your beforeMount hook

Reply:
I dont know if i understand exactly what you are saying, but I tried to give each route a variable that would have been checked in `<v-list-icon v-if="route.variable === isLoggedOrNot">` but the linter screamt at me because you can't put a v-if and a v-for in the same pair, therefore I did the 2 container with copy pasted things

Reply:
find a way to do it without code duplication in 9ac7672